json-server/config/db.js

```diff
@@ -2164,6 +2164,36 @@ module.exports = function () {
                 "label": "Unknown"
             }
         ],
+        "gcmIssueType": [
```

Comment:
Not for this ticket but I have a feeling that if we have the end point in Sirius loads of these route will not be needed for Cypress as we will get the information straight from Sirius 🤔